<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="title">
      <string>File Management</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Ignore Files matching:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="file_ignore_edit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ignore Folders matching:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="folder_ignore_edit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Multiple patterns can be separated by '|', all matching is case insensitive.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="platesolver_group">
     <property name="title">
      <string>Plate Solving Settings</string>
     </property>
     <layout class="QVBoxLayout" name="platesolver_layout">
      <item>
       <layout class="QHBoxLayout" name="astap_layout">
        <item>
         <widget class="QLabel" name="astap_label">
          <property name="text">
           <string>ASTAP Path:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="astap_path_edit">
          <property name="toolTip">
           <string>Path to the ASTAP executable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="astap_browse_button">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="astrometry_layout">
        <item>
         <widget class="QLabel" name="astrometry_label">
          <property name="text">
           <string>Astrometry.net API Key:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="astrometry_api_key_edit">
          <property name="toolTip">
           <string>API key for astrometry.net</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
